src/selectors/markdownSelectors.test.js

```diff
@@ -0,0 +1,19 @@
+import { getMarkdown } from './markdownSelectors';
+
+describe('selector tests', () => {
+  it('test the get single markdown selector', () => {
+    const state = {
+      markdown: 'some string'
+    };
+    const markdown = getMarkdown(state);
+    expect(markdown).toEqual('some string');
+  });
+
+  it('it will return an empty state if state is empty', () => {
```

Comment:
it it will return an empty state if state is empty. maybe:

it gets empty markdown text